src/main/java/core/basesyntax/FigureList.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public enum FigureList {
```

Comment:
```suggestion
public enum Figure {
```
It is not a proper name for an enum. Enum is not a list)